pallets/auctions/src/lib.rs

```diff
@@ -183,9 +199,19 @@ pub mod pallet {
 
 		#[pallet::weight(<T as Config>::WeightInfo::delete_auction())]
 		pub fn delete_auction(origin: OriginFor<T>, id: T::AuctionId) -> DispatchResultWithPostInfo {
```

Comment:
It is laready mixed use of DispatchResult and DispatchResultWithPostInfo in this pallet.


Following refactor #111 , it is actaully better to use DispatchResult unless you need specifically the post info.

Applies to each dispatchable in this pallet.

```suggestion
		pub fn delete_auction(origin: OriginFor<T>, id: T::AuctionId) -> DispatchResult {
```